index.html

```diff
@@ -0,0 +1,150 @@
+                    <h2>Celebrities</h2>
+                    <h3>Famous musicians have going to participate in events all the days. Dont lose this oportunity!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/theme-park-icon.png" alt="" class="card-img">
+                    <h2>Theme Park</h2>
+                    <h3>You can visit diverse atractions all the day!Dont fogot go to the mountain russian!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/sun-icon.png" alt="" class="card-img">
+                    <h2>Non stop!</h2>
+                    <h3>This party dont gonna stop never! so you can be parting 24/7</h3>
+                </div>
+            </div>
+            <div class="button-container">
+            <button class="button-cards" id="join">Join to the party!</button>
+            </div>
+            <div class="grid-container-desktop">
+                <div class="card"><img src="sources/mobile-version/images/mask-icon.png" alt="" class="card-img">
+                    <h2>thematic events</h2>
+                    <h3>Feel free to play as a character in the night of the cartoons, and roleplay in the other special events!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/tickets-icon.png" alt="" class="card-img">
+                    <h2>Free Bar & Entry</h2>
+                    <h3>You can ask for a free drink every day. As well get a free entry to one show in the last day of the event</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/superstar-icon.png" alt="" class="card-img">
+                    <h2>Celebrities</h2>
+                    <h3>Famous musicians have going to participate in events all the days. Dont lose this oportunity!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/theme-park-icon.png" alt="" class="card-img">
+                    <h2>Theme Park</h2>
+                    <h3>You can visit diverse atractions all the day!Dont fogot go to the mountain russian!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/sun-icon.png" alt="" class="card-img">
+                    <h2>Non stop!</h2>
+                    <h3>This party dont gonna stop never! so you can be parting 24/7</h3>
+                </div>
+            </div>
+            <div class="button-container">
+            </div>
+            <a href="" class="link-desktop">SEE THE WHOLE PROGRAM</a>
+        </section>
+        <section class="artists">
+            <div></div>
+            <h2>Featured Artists</h2>
+            <div class="tiny-line"></div>
+            <div class="container-speakers">
+            </div>
+            <div class="hidden-verification"></div>
+            <button class="button-artists">
```

Comment:
- [ ] please as mentioned earlier, the gray pattern background fit well for the mobile version but the desktop version they are away from the image thereby disrupting your design from matching with the original. Please, kindly resolve this. Make if fix behind the image. You can set them as background image for the speaker images.
![Screenshot (1175)_LI](https://github.com/SaveryIV/First-Capstone-Project/assets/65298048/d758787d-ff85-4a04-a0b2-b01ffe7e896c)